front/lib/api/feature_flags.ts

```diff
@@ -4,7 +4,7 @@ import { WHITELISTABLE_FEATURES } from "@dust-tt/types";
 import { isDevelopment } from "@app/lib/development";
 import { FeatureFlag } from "@app/lib/models/feature_flag";
 
-const { ACTIVATE_ALL_FEATURES_DEV = true } = process.env;
```

Comment:
do you intend to switch it back to true after testing?

Reply:
No, this should be in your env if you want it, but having true here is the same as not having this variable to begin with (it's impossible to set it to something falsy since env vars are just text)